cmd/krane/main.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	keychain := authn.NewMultiKeychain(
 		authn.DefaultKeychain,
 		google.Keychain,
+		github.Keychain,
```

Comment:
This was gonna be my feedback.  Now I’m regretting not calling this omicrane after all.

Reply:
It's only been released once as `krane`, you can change it.

Ooooooor, release it as `github.com/mattmoor/omnicrane` instead... 😈